commit b06a1408458cb232009f628ce5bb6177ea724cb7
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Tue Jan 25 04:17:24 2022 +0100

    fix build
    
    For some reason this does not get generated.

diff --git a/help/LINGUAS b/help/LINGUAS
index fc3cd4c..8136caa 100644
--- a/help/LINGUAS
+++ b/help/LINGUAS
@@ -12,7 +12,6 @@ es
 eu
 fr
 gl
-hi
 hu
 id
 it
